src/configuration.cpp

```diff
@@ -161,152 +163,172 @@ chimera::CompiledConfiguration::CompiledConfiguration(
     }
 
     // Get a reference to the configuration YAML root node.
-    // If it is not available, then skip the remaining parsing.
-    if (!configNode_)
-        return;
-
-    // Parse 'namespaces' section of configuration YAML if it exists.
-    const YAML::Node &namespacesNode = configNode_["namespaces"];
-    if (namespacesNode)
+    // If it is not available, then skip configuration node parsing.
+    if (configNode_)
```

Comment:
I didn't actually change any logic here, I just moved the following lines inside the `if` block (up to `L289`), then added the block from `L289` to `L303` where we also check for a `binding: FOO` in the YAML.